public/js/components/ArticleCommissionedLengthTable.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import {Table, Tr, Th, Td} from './Table';
+
+const compare = (article1, article2) => article2.commissionedWordCount - article1.commissionedWordCount;
+
+const ArticleCommissionedLengthTable = ({articles}) => {
+    const noCommissioningLength = articles.filter(article => article.commissionedWordCount == 0)
```

Comment:
Always try and use `===` instead of `==` in JS as javascript has some weird coercion (have a look at "Loose equality using ==" on https://developer.mozilla.org/en-US/docs/Web/JavaScript/Equality_comparisons_and_sameness).

Reply:
👍 fixed